src/OOVPADatabase/D3D8LTCG/4928.inl

```diff
@@ -147,6 +147,33 @@ OOVPA_SIG_MATCH(
     //
 );
 
+// test case: Crimson Sea
```

Comment:
nit: Move below the title symbol comment, not above. testcase is one word for internal documentation.